bitcoin/_posts/0000-00-00-advent-27.md

```diff
@@ -0,0 +1,287 @@
+[here](/bitcoin/2021/12/23/advent-27/) or subscribe at
+[judica.org/join](https://judica.org/join) to get the new posts in your inbox_
+
+
+I know, I know.
+
+God forbid, a roadmap. People hate roadmaps. As I've noted
+[before](/blog/2021/07/02/covenants/):
+
+> **Bitcoin Eschews Roadmaps and Agendas.**
+> I provide this maxim to make clear that this document is by no means an official
+> roadmap, narrative, or prioritization. However, it is my own assessment of what
+> the current most pragmatic approach to upgrading Bitcoin is, based on my
+> understanding of the state of outstanding proposals and their interactions.
+> 
+> My priorities in producing this are to open a discussion on potential new
+> features, risk minimization, and pragmatic design for Bitcoin.
+
+If you didn't click, I definitely recommend reading [the quoted
+post](/blog/2021/07/02/covenants/) in conjunction with this one. As well as, if
+you're a first time visitor to the Advent Calendar, the preceding 26 posts.
+
+<hr>
+
+In contrast to my prior post, this roadmap is going to be less about full
+justifications and unbiased weightings and tallyings of sentiments and more just
+me spitting out a timeline we could introduce changes on. It's not a final
+answer, and in no way authoritative, but it's a launch point for a discussion
+that has to happen in some way at some point in order to advance a soft-fork.
+
+_Consider this as being posted for the sake of public review. If you disagree
+with this, let me know why! But please no attacks just for the act of discussing
+the topic of soft-fork activation[^ordo]._
+
+[^ordo]: You do you though. It'd really hurt my feelings if you called me a Smelly Weasel attacking Bitcoin.
+
+So buckle up here's how we could make Bitcoin kick ass in 2022 and beyond:
+
+# 2022: The Year of the Covenant
+
+## BIP-119 Timeline
+CheckTemplateVerify is getting close to ready to go. There are numerous
+[supporters (listed on utxos.org)](https://utxos.org/signals), few detractors,
+and a bumper crop of amazing use cases (did you read the calendar?) waiting for
+us on the other side of CTV activation. The major critiques are that we might want
+something that does 'more' than CTV itself, or to include it in a bundle of
+things.
+
+My take: we're not Gordon Gecko. Greed is _Not_ Good. CTV represents a clean, well
+contained, unproblematic upgrade that's going to deliver hella functionality in
+service of scaling, decentralization, self custody, and privacy. Let's secure
+the bag for Bitcoin users everywhere and make it happen. We can always do more,
+later, informed by what extensions we need for rapidly maturing tools like
+Sapio. CTV is also technically specified and implemented sufficiently -- a view
+I've confirmed with a couple other devs -- that it is able to be considered for
+release.
+
+### What would have to happen to release CTV?
+
+1. More signalers would need to be on
```

Comment:
There is no list long enough to capture what it would mean to have consensus.

I will add something to the above effect to make clear that a list is only helpful as a strawpoll.